troupon/userprofile/templates/userprofile/profile.html

```diff
@@ -38,7 +38,7 @@
                 <p> Interest:  {{ profile.interest }} </p>
             </div>
 
-            <form action="/account/profile/user/{{ request.user.username }}" method="POST" id="profile-section"><input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
+            <form action="/userprofile/user/{{ request.user.username }}" method="POST" id="profile-section"><input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
```

Comment:
**Code Quality: Nitpick**

You should use `url` here. This makes it easier to reference URLs especially if they are modified eventually.